### DISCUSSION TITLE
getInitialProps Is not working

### DISCUSSION THREAD

**Discussion by gouravsharma07** [vercel/next.js#53511] [Help]
### Summary

I have created client side pages using getStaticProps on my Next js application. 
Now I am creating some server side pages on that using getInitialProps but this is not working , because I am using 'next export' in my package.json.
Anyone  worked on  that type of issue ? 

Showing error in console Error: Export encountered errors on following paths: /content-publishing-system

**Comment by edsphinx** [comment]
You are using Pages Router or App Router? 
Becasue `getInitialProps getStaticProps getServerSideProps` only works with Pages Routing, you can check this on the docs:
https://nextjs.org/docs/pages/building-your-application/data-fetching
and for data fetching with app router:
https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating
